test/y2storage/proposal/autoinst_size_parser_test.rb

```diff
@@ -69,7 +69,11 @@
           expect(size_info.min).to eq(128.MiB)
           expect(size_info.max).to eq(1.GiB)
           expect(size_info.percentage).to be_nil
-          expect(size_info.weight).to be_nil
+        end
+
+        it "sets weight according to control file value" do
```

Comment:
So we stopped testing the case with `"weight" => nil` in order to start testing `weight != nil`? Why are we replacing the test instead of adding one?